website/layouts/_default/index.json.json

```diff
@@ -0,0 +1,5 @@
+{{- $data := newScratch -}}
```

Comment:
Is the double `.json` on the file name intentional?

Reply:
Yeah, that's just how Hugo's templating system works